public/webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require("path");
+const webpack = require("webpack");
+
+module.exports = {
+  entry: { "./src/index": "./src/index.ts", "./src/panel": "./src/panel.ts" },
```

Comment:
What do "./src/index" and "./src/panel" do?  Not sure why we have those in addition to the .ts versions.

Reply:
I think that this map determines where to output the bundle/build for each file.  So, if I'm understanding correctly,  the {`./src/index: ./src/index.ts`} is telling Webpack to output the bundle for `./src/index.ts` into `./src/index.js`

Reply:
Whoops! I totally misread this syntax and at a glance thought it was an array of 4 values rather than a map.  Makes sense now, thanks!